deployment/terraform/assets/dashboard_data.json

```diff
@@ -653,13 +653,22 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "increase(mattermost_http_request_duration_seconds_sum{instance=~\"$server\"}[5m]) /   increase(mattermost_http_request_duration_seconds_count{instance=~\"$server\"}[5m])",
+          "expr": "rate(mattermost_http_request_duration_seconds_sum{instance=~\"$server\"}[5m]) /   rate(mattermost_http_request_duration_seconds_count{instance=~\"$server\"}[5m])",
```

Comment:
These should be "increase". And also, while we're at it, please decrease the time interval to 1m.